crates/relayer-cli/src/commands/create/channel.rs

```diff
@@ -322,6 +323,9 @@ impl CreateChannelCommand {
     ) {
         let config = app_config();
 
+        // Set global registry to get or spawn chain handles
+        set_global_registry(SharedRegistry::new((*app_config()).clone()));
```

Comment:
Even though the registry is being set here (as it is needed further down in the chain of function calls), the chains are being spawned using `spawn_chain_runtime()` rather than using `get_or_spawn()` within `run_multihop_reusing_connection()`. This was to keep it consistent with `run_reusing_connection()` which was already implemented and uses `spawn_chain_runtime()`. 
As `get_or_spawn()` might give us some performance, if we decide to keep using the registry in the CLIs we should use it here and make it consistent across all methods.

Reply:
Good catch! Yeah let's see by the end of the implementation phase which technique we want to go with.